blinkit-app/src/homePage/atoms/insertSelect/InsertSelect.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import { UserConsumer } from "../../../userContext";
+
+const addOption = (value, index) => {
+  if (index === 0)
+    return (
+      <option key={"option-" + index} value={value} hidden>
+        {value}
+      </option>
+    );
+  return (
+    <option key={"option-" + index} value={value}>
+      {value}
+    </option>
+  );
+};
+
+export class InsertSelect extends Component {
+  render() {
+    const values = this.props.values;
+
+    return (
+      <UserConsumer>
+        {(data) => {
+          console.log( "data is ",data )
+          return (
+            <select
+              name={this.props.name}
+              id={this.props.id}
+              onChange={data.filterHandler}>
```

Comment:
filterHandler need not be in the context consumer, create a helper file for this **molecule** and import it here and use it as a default prop value.